pkg/pod/script.go

```diff
@@ -61,8 +81,9 @@ func convertScripts(shellImage string, steps []v1beta1.Step, sidecars []v1beta1.
 		VolumeMounts: []corev1.VolumeMount{scriptsVolumeMount},
 	}
 
-	convertedStepContainers := convertListOfSteps(steps, &placeScriptsInit, &placeScripts, "script")
+	var convertedStepContainers []corev1.Container
```

Comment:
It looks like this could just be moved so it's declared below on line 103: `convertedStepContainers := convertListOfSteps(...)`

Reply:
Yes you're right